plugins/module_utils/ece.py

```diff
@@ -188,16 +183,29 @@ def update_deployment_info(self, deployment_name, config):
         break
     return target_deployment_object
 
-  def create_cluster(self):
+  def create_cluster(self, 
+                     cluster_name, 
+                     version, 
+                     deployment_template, 
+                     elastic_settings, 
+                     kibana_settings, 
+                     elastic_user_settings, 
+                     apm_settings, 
+                     ml_settings = None, 
+                     snapshot_settings = None, 
+                     traffic_rulesets = None,
+                     wait_for_completion = False,
+                     completion_timeout = 600
+                     ):
```

Comment:
not going to block approval, but I'm curious why the change from using the class's settings over to specifying everything explicitly like this.

Reply:
It is more that you would call ECE and not be creating an elastic instance.  for instance terminating it. We may very well do other things that wouldn't use those variables as well